### PR TITLE
Added official Travis C# support

### PR DESCRIPTION
**Pull request by Mailaender**
Test of https://github.com/travis-ci/travis-build/pull/294.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,7 @@
...
-language: c
-
-# Make sure build dependencies are installed.
-install:
- - wget http://download.mono-project.com/repo/xamarin.gpg -O /tmp/mono.gpg
- - sudo apt-key add /tmp/mono.gpg
- - sudo sh -c "echo 'deb http://download.mono-project.com/repo/debian wheezy/snapshots/3.8.0 main' >> /etc/apt/sources.list.d/mono-xamarin.list"
- - sudo apt-get update -qq
- - sudo apt-get install -qq mono-gmcs cli-common-dev libgdiplus=2.10-3
-cache: apt
```
Comment (simi) [inline .travis.yml:14]: Why `cache` is removed?

Comment (Mailaender) [inline .travis.yml:14]: I thought it is redundant now.

Comment (simi) [inline .travis.yml:14]: https://staging.travis-ci.org/Mailaender/OpenRA/caches

Comment (Mailaender) [inline .travis.yml:14]: https://github.com/OpenRA/OpenRA/issues/7073 is the production pull request you will want to review by the way. Looks like the cache feature never worked: https://travis-ci.org/OpenRA/OpenRA/caches is also empty.

**Pull request closed without merging**